addons/web/static/src/core/domain_selector_dialog/domain_selector_dialog.js

```diff
@@ -95,7 +94,6 @@ DomainSelectorDialog.props = {
     resModel: String,
     className: { type: String, optional: true },
     defaultConnector: { type: [{ value: "&" }, { value: "|" }], optional: true },
-    defaultLeafValue: { type: Array, optional: true },
```

Comment:
maybe we could move all of these properties in the class as static properties

Reply:
done